### ISSUE TITLE
Could you incorporate a functionality that allows using a connected device as an input device for the PC?

### ISSUE THREAD

**Issue by abhaystar2004** [Genymobile/scrcpy#4815]
**Enhanced Productivity:**
Imagine having your tablet or smartphone act as a secondary input device alongside your keyboard and mouse. You can use it for quick tasks like navigating through presentations, controlling media playback, or even as a virtual touchpad.
For graphic designers and artists, using a tablet with stylus support as an input device provides a more natural and precise way to create digital art or edit images.
**Flexible Interaction:**
When your primary input devices (keyboard and mouse) are occupied, having an additional device as an input option can be incredibly useful. For instance:
During video conferences, you can use your tablet to mute/unmute, adjust volume, or switch between shared screens without interrupting the conversation.
While gaming, you can map specific controls to your tablet, freeing up your keyboard for other functions.
**Accessibility and Ergonomics:**
Some users may find it more comfortable to use a touchscreen device for certain tasks. For individuals with mobility issues, using a tablet or smartphone as an input device can be a game-changer.
It allows users to interact with the PC in a way that suits their preferences and physical capabilities.
**Collaboration and Presentation:**
In collaborative settings, such as meetings or workshops, you can use your tablet to annotate documents, draw diagrams, or highlight key points during presentations.
Teachers can write on a tablet while sharing their screen, making online classes more engaging.
**Creative Workflows:**
Artists, architects, and designers often prefer drawing directly on a tablet. By integrating tablet input, they can seamlessly switch between sketching on the tablet and working on their PC.
Using a tablet as a graphics tablet (similar to a Wacom device) allows for precise control over digital art applications.

Also there is no such tool to do that efficiently

**Comment by risharde** [comment]
I use an app called TouchPortal for what you mentioned - it's not free but it's cheap. I only mention this because scrcpy seems like more of a pc to android control app and you want something the other way around correct?

**Comment by abhaystar2004** [comment]
Yeah, I wanted the other way around so that I could use my tablet as a drawing pad. TouchPortal uses wireless connectivity. I was looking for a wired connection for better connectivity, low latency, and ease of use.

**Comment by hwf1324** [comment]
I use SpaceDesk to do something similar, although it is only equivalent to an extended touchscreen. I use a wired Ethernet connection, maybe also support USB transfer? This is much faster than wireless connection.
